circuits/circom/utils/non-qv/messageValidator.circom

```diff
@@ -46,6 +46,10 @@ template MessageValidatorNonQv() {
 
     // True when the command is valid; otherwise false.
     signal output isValid;
+    // True if the state leaf index is valid or not
```

Comment:
I think it should be `The state leaf index is valid or not` or `True if the state leaf index is valid` ?
I mean just choose either `true` or `or not` not both?